.github/workflows/release.yml

```diff
@@ -109,8 +113,14 @@ jobs:
         name: 'Prepare CDN release'
         run: echo "versions=$(npm run --silent ci:prepare-release)" >> ${GITHUB_OUTPUT}
 
+      - name: generate build provenance
+      if: ${{ hashFiles('packages/rum/dist/bundles/*.js') }}
```

Comment:
```suggestion
        if: ${{ always() && hashFiles('packages/rum/dist/bundles/*.js')
```

Reply:
I'm not actually sure if and why this is required. Just saw in the subsequent steps